### PR TITLE
Feature/readme

### PR DESCRIPTION
**Pull request by Pierozi**
Address #6 
- [x] Create readme based on last Hoa General Readme instruction.
- [x] Implement quick usage with easy example.
- [x] Text description of library.

Show readme file here : 
https://github.com/Pierozi/HoaHeap/blob/feature/readme/README.md

### CODE REVIEW

**Comment by Pierozi** [comment]
@jubianchi I saw on irc you are interested in Lib. Review and comments about Code/Doc/Api... if you want.

File: README.md [hunk 1]
```diff
@@ -1 +1,188 @@
...
+This library provides a set of advanced Heap can support Scalar, Array, Object or Closure
+as item and not only Integer, as ordinal does.
+The order of heap depends of priority parameter.
+
+`Hoa\Heap\Min` and `Hoa\Heap\Max` class interpret priority by comparing items numerically.
+But you are free to implement your own class if you want a different sort algorithm.
+
+## :warning: Warning
+The default iteration process do not enqueue the Heap as common usage.
+You must use Generator methods `pop` or `pop` for iterate on with remove item from heap.
```
Comment (Pierozi) [inline README.md]: CS => `pop` or **`top`**

File: README.md [hunk 2]
```diff
@@ -1 +1,188 @@
...
+```sh
+$ composer install
+$ vendor/bin/hoa test:run
+```
+
+For more information, please consult the [contributor
+guide](https://hoa-project.net/Literature/Contributor/Guide.html).
+
+## Quick usage
+
```
Comment (Pierozi) [inline README.md]: Quick usage use method as Heap value, but any kind of value can be specified, Scalar, Object, Resources...

File: README.md [hunk 3]
```diff
@@ -1 +1,188 @@
...
+This library provides a set of advanced Heap can support Scalar, Array, Object or Closure
+as item and not only Integer, as ordinal does.
+The order of heap depends of priority parameter.
+
+`Hoa\Heap\Min` and `Hoa\Heap\Max` class interpret priority by comparing items numerically.
+But you are free to implement your own class if you want a different sort algorithm.
+
+## :warning: Warning
+The default iteration process do not enqueue the Heap as common usage.
```
Comment (jubianchi) [inline README.md]:
``` diff
- do not enqueue
+ do not dequeue
```

Comment (Pierozi) [reply README.md]: Thanks :+1:

**Pull request merged**